ocaml/database/redo_log.ml

```diff
@@ -888,6 +888,26 @@ let flush_db_to_redo_log db log =
     write_db_to_fd log ;
   !(log.currently_accessible)
 
+let flush_db_or_raise db log =
```

Comment:
We have used `exn` as suffix to indicate a function may fail. I like `or_raise` but it's a bit longer.